Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>plugins {\r\n    id 'java'\r\n    id 'io.qameta.allure' version '2.11.2'\r\n}\r\n\r\ngroup = 'brezitskaya'\r\nversion = '1.0-SNAPSHOT'\r\n\r\nrepositories {\r\n    mavenCentral()\r\n}\r\n\r\ndependencies {\r\n    testImplementation platform('org.junit:junit-bom:5.9.1')\r\n    testImplementation 'org.junit.jupiter:junit-jupiter:5.9.1'\r\n    implementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'\r\n    implementation 'io.qameta.allure:allure-junit5:2.25.0'\r\n    implementation 'org.seleniumhq.selenium:selenium-java:4.31.0'\r\n    testImplementation 'org.assertj:assertj-core:3.25.3'\r\n}\r\n\r\ntest {\r\n    useJUnitPlatform()\r\n}\r\n\r\ntasks.register('smokeTest', Test) {\r\n    useJUnitPlatform() {\r\n        includeTags 'smoke', 'api'\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision f2ec7ec22010faaf8ce1ec647e7481d3ac35359b)
+++ b/build.gradle	(date 1750016787203)
@@ -16,7 +16,7 @@
     implementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'
     implementation 'io.qameta.allure:allure-junit5:2.25.0'
     implementation 'org.seleniumhq.selenium:selenium-java:4.31.0'
-    testImplementation 'org.assertj:assertj-core:3.25.3'
+    implementation 'org.assertj:assertj-core:3.25.3'
 }
 
 test {
